app/(gcforms)/[locale]/(form administration)/form-builder/[id]/components/dialogs/MoreDialog/MoreDialog.tsx

```diff
@@ -103,7 +108,14 @@ export const MoreDialog = () => {
           title={t("moreOptions")}
```

Comment:
We should move this out to a const -- t("moreOptions")

Minor but no sense re-grabbing the string every render